### PR TITLE
Add etcd

### PR DESCRIPTION
**Pull request by shpyrkayevhen**
Add etcd db to the project.

### CODE REVIEW

File: db.py [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+    @abstractmethod
+    def get(self, key): pass
+
+    @abstractmethod
+    def delete(self, key): pass
+        
+
+class EtcdConnect(Db):
+    '''Actions with etcd db.'''
```
Comment (oshynk) [inline db.py]: If I'm not mistaken PEP says that docstrings should be in triple double quotes. please check this

File: db.py [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+        self._host = host
+        self._port = port
+
+    def _connect(self):
+        '''Connect to etcd db.'''
+        self._instance = etcd3.client(host=self._host, port=self._port)
+
+    def write(self, key: str, value) -> None: 
+        '''Write data to db.'''
+        self._connect()
```
Comment (oshynk) [inline db.py]: I think you can just create client once and do not recreate it for each operation call. Check project code for an example

File: db.py [hunk 3]
```diff
@@ -0,0 +1,45 @@
+from abc import ABC, abstractmethod
+
+import etcd3 
+
+
+class Db(ABC):
+    '''Abstract class for db.'''
+    @abstractmethod
+    def write(self, key=None, value=None): pass
```
Comment (oshynk) [inline db.py]: Why did you decide to have default values here?

File: server.py [hunk 4]
```diff
@@ -43,10 +41,9 @@ def getUsers(self, request: chat_pb2.getUsersRequest, context) -> chat_pb2.getMe
 
     def getMessages(self, request: chat_pb2.getMessagesRequest, context) -> chat_pb2.getMessagesResponce:
         """Returns all messages particular user."""
-        user_login = request.user.login
-        if user_login in messages:
-            for message in messages[user_login]:
-                yield chat_pb2.getMessagesResponce(message=message)
+        for messages in etcd.get(f'{request.user.login}'):
```
Comment (oshynk) [inline server.py:47]: Maybe we can delete message from db once it's received?

File: db.py (hunk 3 above)
Comment (shpyrkayevhen) [reply db.py]: Be honest, I don't remember 🙈 But u are right, I also don't see any reasons to use it there!

**Review by oshynk — APPROVED** [review]

**Pull request merged**